app/pages/neworg/findorg/contextCreator.js

```diff
@@ -1,12 +1,21 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../config';
 
-export const getContext = ({ odsCode }) => ({
+const getErrorMessage = ({ errorCode }) => {
+  let message;
+  if (errorCode === '404') message = 'Organisation not found';
+  if (errorCode === '406') message = 'Not a buyer organisation';
```

Comment:
we can have a little object map that we can use to get the message out. This will allows us to add more to it without having more `if` clauses in here